docs/krel/gcbmgr.md

```diff
@@ -0,0 +1,183 @@
+# gcbmgr (GCB Manager) <!-- omit in toc -->
+
+`gcbmgr` is a subcommand of `krel` for Release Managers to submit Kubernetes staging and release jobs to Google Cloud Build (GCB).
+
+`krel gcbmgr` is a wrapper around the `gcloud builds submit` command which passes a set of GCB substitutions (variables) to a GCB configuration for usage in [`anago`](/anago).
+
+**NOTE: `krel gcbmgr` is currently in development and its design may rapidly change. If you encounter errors, please file an issue in this repo.**
+
+- [Installation](#installation)
+- [Usage](#usage)
+- [Important notes](#important-notes)
+- [Alpha](#alpha)
+  - [Alpha Stage](#alpha-stage)
+  - [Alpha Release](#alpha-release)
+- [Beta](#beta)
+  - [Beta Stage](#beta-stage)
+    - [Branch cut (`x.y.0-beta.0`)](#branch-cut-xy0-beta0)
+    - [Post-branch cut (`x.y.0-beta.1` and beyond)](#post-branch-cut-xy0-beta1-and-beyond)
+  - [Beta Release](#beta-release)
+- [Release Candidate](#release-candidate)
+  - [Release Candidate (RC) Stage](#release-candidate-rc-stage)
+  - [Release Candidate (RC) Release](#release-candidate-rc-release)
+- [Official](#official)
+  - [Official Stage](#official-stage)
+  - [Official Release](#official-release)
+
+## Installation
+
+From this root of this repo:
+
+```shell
+./compile-release-tools krel
```

Comment:
What is the correct way? because we can run `make release-tools` as well right?

Reply:
I think both would be fine :)

Reply:
Yep, either is fine!